modules/ROOT/pages/overview-cli-extensions.adoc

```diff
@@ -0,0 +1,36 @@
+= Using Anypoint Platform DataGraph CLI and API Extensions
+
+You can use the Anypoint CLI DataGraph commands to add and edit API sources in DataGraph. To do so you must first add the appropriate DataGraph API extensions to the API source. 
+
+== Adding Extensions to Your Project
+
+The Datagraph API extensions are available as a public asset in Exchange[link???]. You must add the extensions to your API-specification project as a dependency, and then publish the project to Exchange. After you publish your API-specification project, you can add the API to DataGraph using the CLI.
```

Comment:
The extension can't be viewed in Exchange (hence the lack of a link) and is only available as a fragment in API Designer.

Reply:
```suggestion
You must add the extensions to your API-specification project as a dependency, and then publish the project to Exchange. After you publish your API-specification project, you can add the API to DataGraph using the CLI.
```

Reply:
@Extor3D is this correct then?